### ISSUE TITLE
[Bug]: sd-webui-v1.6.0-127 Controlnet v1.1.410 TypeError: unhashable type: 'list'

### ISSUE THREAD

**Issue by meacial** [Mikubill/sd-webui-controlnet#2126]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

Traceback (most recent call last):
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/gradio/routes.py", line 488, in run_predict
    output = await app.get_blocks().process_api(
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/gradio/blocks.py", line 1431, in process_api
    result = await self.call_function(
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/gradio/blocks.py", line 1103, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/anyio/to_thread.py", line 33, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 877, in run_sync_in_worker_thread
    return await future
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 807, in run
    result = context.run(func, *args)
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/gradio/utils.py", line 707, in wrapper
    response = f(*args, **kwargs)
  File "/local/xxx/temp/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/controlnet_ui/controlnet_ui_group.py", line 600, in filter_selected
    ) = global_state.select_control_type(k)
  File "/local/xxx/temp/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/global_state.py", line 242, in select_control_type
    default_option = preprocessor_filters[control_type]
TypeError: unhashable type: 'list'
Traceback (most recent call last):
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/gradio/routes.py", line 488, in run_predict
    output = await app.get_blocks().process_api(
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/gradio/blocks.py", line 1431, in process_api
    result = await self.call_function(
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/gradio/blocks.py", line 1103, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/anyio/to_thread.py", line 33, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 877, in run_sync_in_worker_thread
    return await future
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 807, in run
    result = context.run(func, *args)
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/gradio/utils.py", line 707, in wrapper
    response = f(*args, **kwargs)
  File "/local/xxx/temp/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/controlnet_ui/controlnet_ui_group.py", line 600, in filter_selected
    ) = global_state.select_control_type(k)
  File "/local/xxx/temp/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/global_state.py", line 242, in select_control_type
    default_option = preprocessor_filters[control_type]
TypeError: unhashable type: 'list'

### Steps to reproduce the problem

1. sd-webui-v1.6.0-127 Controlnet v1.1.410 
2. using controlnet , select "Control Type" 
3.  Always loading 
4. has error in log.

### What should have happened?

ssss

### Commit where the problem happens

webui:  1.6.0
controlnet:  1.1.40

### Command Line Arguments

```Shell
no
```

### List of enabled extensions

controlnet 

### Console logs

```Shell
Traceback (most recent call last):
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/gradio/routes.py", line 488, in run_predict
    output = await app.get_blocks().process_api(
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/gradio/blocks.py", line 1431, in process_api
    result = await self.call_function(
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/gradio/blocks.py", line 1103, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/anyio/to_thread.py", line 33, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 877, in run_sync_in_worker_thread
    return await future
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 807, in run
    result = context.run(func, *args)
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/gradio/utils.py", line 707, in wrapper
    response = f(*args, **kwargs)
  File "/local/xxx/temp/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/controlnet_ui/controlnet_ui_group.py", line 600, in filter_selected
    ) = global_state.select_control_type(k)
  File "/local/xxx/temp/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/global_state.py", line 242, in select_control_type
    default_option = preprocessor_filters[control_type]
TypeError: unhashable type: 'list'
Traceback (most recent call last):
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/gradio/routes.py", line 488, in run_predict
    output = await app.get_blocks().process_api(
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/gradio/blocks.py", line 1431, in process_api
    result = await self.call_function(
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/gradio/blocks.py", line 1103, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/anyio/to_thread.py", line 33, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 877, in run_sync_in_worker_thread
    return await future
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 807, in run
    result = context.run(func, *args)
  File "/root/anaconda3/envs/webuisd/lib/python3.10/site-packages/gradio/utils.py", line 707, in wrapper
    response = f(*args, **kwargs)
  File "/local/xxx/temp/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/controlnet_ui/controlnet_ui_group.py", line 600, in filter_selected
    ) = global_state.select_control_type(k)
  File "/local/xxx/temp/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/global_state.py", line 242, in select_control_type
    default_option = preprocessor_filters[control_type]
TypeError: unhashable type: 'list'
```

### Additional information

ss

**Comment by UncleK** [comment]
I encountered this too, deleting venv folder didn't help,
only happened in "selecting "Control Type" ", manually selections of Preprocessor and models are okay.

**Comment by derekhsu** [comment]
Hi, I have the same issue. Is there any updated information?

**Comment by huchenlei** [comment]
See https://github.com/Mikubill/sd-webui-controlnet/issues/2017

TL;DR is update your A1111 version.
